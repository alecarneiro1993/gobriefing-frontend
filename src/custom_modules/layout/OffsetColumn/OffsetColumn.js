// @flow

import React from 'react';
import Grid from '@material-ui/core/Grid';

type Props = {};

export default function OffsetColumn(props: Props) {
  return <Grid item {...props} />;
}
