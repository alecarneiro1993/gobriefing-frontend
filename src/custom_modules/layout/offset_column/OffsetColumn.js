import React from "react";
import { shallow } from "enzyme";

import OffsetColumn from ".";

describe("OffsetColumn", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<OffsetColumn />);
    expect(wrapper).toMatchSnapshot();
  });
});
