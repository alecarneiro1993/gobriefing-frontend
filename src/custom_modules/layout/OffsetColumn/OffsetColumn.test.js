import React from 'react';
import { shallow } from 'enzyme';

import OffsetColumn from './OffsetColumn';

describe('OffsetColumn', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<OffsetColumn />);
    expect(wrapper).toMatchSnapshot();
  });
});
