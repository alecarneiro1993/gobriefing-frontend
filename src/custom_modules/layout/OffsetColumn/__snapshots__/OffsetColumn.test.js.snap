// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffsetColumn renders without crashing 1`] = `
<WithStyles(ForwardRef(Grid))
  item={true}
/>
`;
